contract-typespec/api/main.tsp

```diff
@@ -0,0 +1,33 @@
+// APIs
+import "./clusters.tsp";
+import "./brokers.tsp";
+import "./topics.tsp";
+import "./messages.tsp";
+import "./consumer-groups.tsp";
+import "./schemas.tsp";
+import "./kafka-connect.tsp";
+import "./ksql.tsp";
+import "./acls.tsp";
+import "./quotas.tsp";
+import "./auth.tsp";
+import "./config.tsp";
+import "./graphs.tsp";
+import "./prometheus.tsp";
+
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using Http;
+using Rest;
+
+@service(#{ title: "Kafbat UI Api Service" })
+@info(#{
+  contact: #{ name: "API Support", email: "support@kafbat.io" },
```

Comment:
I'd create a separate alias for this due to spam reasons, at least different channels

Reply:
let's keep this one as public